cartridge/remote-control.lua

```diff
@@ -63,6 +63,30 @@ local function is_callable(fun)
     end
 end
 
+-- rawget implementation that respects metatable methods
```

Comment:
The patch doesn't correspond to the PR caption "adapt tests for strict mode". It would be better if remote-control module remained unchanged.

Reply:
But remote_control test suite can't be launched in strict mode without this change.

Reply:
Ok

I've disabled strict mode for remote_control tests. It seems one way to provide equal test messages for all Tarantool builds